### ISSUE TITLE
Warning: sprintf(): Too few arguments in \wp-admin\includes\class-bulk-upgrader-skin.php on line 154

### ISSUE THREAD

**Issue by DKudleichuk** [thomasgriffin/TGM-Plugin-Activation#686]
Hello!

Is there any fix please? Wordpress 4.8.1, TGM 2.6.1

![2017-08-18_16-20-56](https://user-images.githubusercontent.com/12991525/29460612-46441d14-8431-11e7-8e00-1a602ad65b15.png)

**Comment by jrfnl** [comment]
Closing as duplicate of #669 which was fixed by PR #670. The fix will be in the next release of TGMPA.

**Comment by saleeeem** [comment]
i'm not understanding

**Comment by Progressed36** [comment]
It says its fixed in next release, but after almost a year there is no new release?

**Comment by Remzi1993** [comment]
There's still no fix.

**Comment by viettamus** [comment]
You can fix it by replacing code in the file: www/wp-admin/includes/class-bulk-upgrader-skin.php
Old code:

```
if ( $this->result && ! is_wp_error( $this->result ) ) {
	if ( ! $this->error ) {
		echo '<div class="updated js-update-details" data-update-details="progress-' . esc_attr( $this->upgrader->update_current ) . '">' .
			'<p>' . sprintf( $this->upgrader->strings['skin_update_successful'], $title ) .
			' <button type="button" class="hide-if-no-js button-link js-update-details-toggle" aria-expanded="false">' . __( 'Show details.' ) . '</button>' .
			'</p></div>';
	}

	echo '<script type="text/javascript">jQuery(\'.waiting-' . esc_js($this->upgrader->update_current) . '\').hide();</script>';
}
```

New code:

```
if ( $this->result && ! is_wp_error( $this->result ) ) {
	if ( ! $this->error )
		echo '<div class="updated"><p>' . sprintf($this->upgrader->strings['skin_update_successful'], $title, 'jQuery(\'#progress-' . esc_js($this->upgrader->update_current) . '\').toggle();jQuery(\'span\', this).toggle(); return false;') . '</p></div>';
	echo '<script type="text/javascript">jQuery(\'.waiting-' . esc_js($this->upgrader->update_current) . '\').hide();</script>';
}
```

**Comment by MrShabir** [comment]
@viettamus Its not a solution, Modify WordPress Core files are not a best solution.

**Comment by CreativeDive** [comment]
When will this fix be released in the new version?

**Comment by magicstickuk** [comment]
Still no proper fix?

**Comment by andyg2** [comment]
Still no fix in 6.1 😕
